Dockerfile.semantic-release

```diff
@@ -0,0 +1,13 @@
+FROM jfloff/alpine-python:2.7-slim
+RUN apk update
+RUN apk add --no-cache bash git
+RUN /entrypoint.sh \
+  -a git \
+  -p python-semantic-release \
+&& echo
+RUN apk add pcre git
+RUN git config --global user.email semantic-release@masschallenge.org
+RUN mkdir /app
+WORKDIR /app
+ADD setup.cfg .
```

Comment:
would've preferred the `setup.cfg` would have a name that identifies it as config for semantic release, and not for any of the many other things that might require a config file around here :)

Reply:
unfortunately this is something that the library we use requires and is not changeable.